app/index.js

```diff
@@ -9,98 +9,60 @@ class BlockstackGenerator extends Generator {
-      'Welcome to the ' + chalk.red('Blockstack') + ' app generator!'
-    ));
+  async prompting() {
+    if (this.options.react) {
+      this.outputAppType = 'react';
+    } else if (this.options.plain) {
+      this.outputAppType = 'plain';
+    } else if (this.options.vue) {
+      this.outputAppType = 'vue';
+    } else {
 
-    var prompts = [{
-      type: 'confirm',
-      name: 'someAnswer',
-      message: 'Are you ready to build a Blockstack app?',
-      default: true
-    }];
+      // Have Yeoman greet the user.
+      this.log(yosay(
+        'Welcome to the ' + chalk.red('Blockstack') + ' app generator!'
+      ));
 
-    return this.prompt(prompts).then(function (props) {
-      // To access props later use this.props.someAnswer
-      this.props = props;
-    }.bind(this));
+      const answers = await this.prompt([{
+        type: 'list',
+        name: 'generator_type',
+        message: 'Select Blockstack app output type',
+        choices: [
+          { name: 'React', value: 'react' },
+          { name: 'Plain JavaScript', value: 'plain' },
+          { name: 'Vue', value: 'vue' }
+        ]
+      }]);
+      this.outputAppType = answers['generator_type'];
+    }
   }
 
-  writing() {
-    this.fs.copy(
-      this.templatePath('_package.json'),
-      this.destinationPath('package.json')
-    );
-    this.fs.copy(
-      this.templatePath('editorconfig'),
-      this.destinationPath('.editorconfig')
-    );
-    this.fs.copy(
-      this.templatePath('gitignore'),
-      this.destinationPath('.gitignore')
-    );
-    this.fs.copy(
-      this.templatePath('requires.js'),
-      this.destinationPath('requires.js')
-    );
-    this.fs.copy(
-      this.templatePath('firebase.json'),
-      this.destinationPath('firebase.json')
-    );
-    // server
-    this.fs.copy(
-      this.templatePath('server.js'),
-      this.destinationPath('server.js')
-    );
-    // styles
-    this.fs.copy(
-      this.templatePath('public/app.css'),
-      this.destinationPath('public/app.css')
-    );
-    this.fs.copy(
-      this.templatePath('public/bootstrap.min.css'),
-      this.destinationPath('public/bootstrap.min.css')
-    );
-    // scripts
-    this.fs.copy(
-      this.templatePath('public/app.js'),
-      this.destinationPath('public/app.js')
-    );
-    // images
-
-    this.fs.copy(
-      this.templatePath('public/favicon.ico'),
-      this.destinationPath('public/favicon.ico')
-    )
-
-    this.fs.copy(
-      this.templatePath('public/index.html'),
-      this.destinationPath('public/index.html')
-    );
-    // publicExtras
-    this.fs.copy(
-      this.templatePath('public/robots.txt'),
-      this.destinationPath('public/robots.txt')
-    );
-    this.fs.copy(
-      this.templatePath('public/manifest.json'),
-      this.destinationPath('public/manifest.json')
-    );
-    this.fs.copy(
-      this.templatePath('public/white-logo.svg'),
-      this.destinationPath('public/white-logo.svg')
-    );
+  main() {
```

Comment:
```
I'm all done. Running npm install for you to install the required dependencies. If this fails, try running the command yourself.
```

Instead of "I'm" 

```
Your PROJECT_TYPE successfully generated. Now running "npm install" the command to install the required dependencies. If the command fails, try running "npm install" yourself.
```

Sub out PROJECT_TYPE with whatever type they selected. If you can avoid the quotes around `npm install` and use tics/fonts to distinguish `npm install` that would be helpful with readability.  There are users who are unfamiliar with `npm` so good to make it stand out.

Reply:
This is existing messaging from `yo`, unchanged by this PR. If we want to modify these messages we should create an issue and prioritize it.

Reply:
Ugh, definitely think they should allow for customizing that message on the dev.  It isn't worth an entirely extra PR to fix.